proto/otaclient_iot_logging_server_v1.proto

```diff
@@ -14,38 +14,50 @@
 
 syntax = "proto3";
 
-service OtaClientLoggingService {
+service OtaClientIoTLoggingService {
   /*
    * `PutLog` service requests OTA Client logging service to put log.
    */
   rpc PutLog(PutLogRequest) returns (PutLogResponse) {}
+
+  /*
+   * `Check` requests OTA Client logging service to check the health of the
+   * service.
+   */
+  rpc Check(HealthCheckRequest) returns (HealthCheckResponse);
 }
 
 enum LogType {
   LOG = 0;
   METRICS = 1;
 }
-enum LogLevel {
-  UNSPECIFIC = 0;
-  TRACE = 1;
-  DEBUG = 2;
-  INFO = 3;
-  WARN = 4;
-  ERROR = 5;
-  FATAL = 6;
-}
+
 enum ErrorCode {
-  OK = 0;
-  FAILED = 1;
+  NO_FAILURE = 0;         // Success
+  SERVER_QUEUE_FULL = 1;  // Error: Server queue is full
+  NOT_ALLOWED_ECU_ID = 2; // Error: Specified ECU ID is not allowed
+  NO_MESSAGE = 3;         // Error: No message in the request
```

Comment:
Still as protobuf will assign 0 to unset field, I recommend to leave 0 as UNSPECIFIC.

Reply:
Thank you, set `UNSPECIFIC_ERROR_CODE` as 0.